packages/main/src/Bar.ts

```diff
@@ -109,11 +128,15 @@ class Bar extends UI5Element {
 
 	get accInfo() {
 		return {
-			"label": this.design,
+			"label": this.ariaLabelText,
 			"role": this.effectiveRole,
 		};
 	}
 
+	get ariaLabelText(): string | undefined {
+		return this.accessibleName ? getEffectiveAriaLabelText(this) : this.design;
```

Comment:
Hi @tsanislavgatev I happened to look at that change
and I thought shouldn't be `accessibleNameRef` taken into account,  like

```ts
if (this.accessibleName || this.accessibleNameRef) {
			return getEffectiveAriaLabelText(this);
}
return this.design;
```

Also, shouldn't be `this.design` i18n-ed?